core/src/main/java/org/jdbi/v3/core/statement/ParsedSql.java

```diff
@@ -72,6 +72,25 @@ public String toString() {
             + '}';
     }
 
+    /**
+     * A static factory for {@link ParsedSql} object. It takes two
+     * arguments. The first one is SQL which must contain only
+     * positional parameters (i.e. the {@value #POSITIONAL_PARAM}
+     * characters). If your code contains named parameters (i.e.
+     * the ones followed by a colon) then you have to replace them
+     * by positional parameters and store mapping as
```

Comment:
I'd reword this to something like
"and specify the mapping in the {@code parameters} (see _links to factory methods here_). You cannot mix named and positional parameters in one SQL statement."

Reply:
Will do.

Reply:
Done.